lib/domain/libBlockchain/langs/yul/YulModel.h

```diff
@@ -189,6 +173,20 @@ namespace vanguard {
             }
         };
 
+        bool callNameMatches(const CallIns<Domain> &call , const std::regex &reg) const {
+            auto unwrapped = call.unwrap();
+            if(const llvm::CallInst *unwrappedCall = llvm::dyn_cast<llvm::CallInst>(unwrapped)) {
+                auto calledFn = unwrappedCall->getCalledFunction();
+                if(!calledFn->hasName()) {
+                    return false;
+                }
+
+                std::string name = calledFn->getName().str();
+                return regex_match(calledFn->getName().str(), reg);
```

Comment:
nit: looks like a typo here
```suggestion
                return regex_match(name, reg);
```